packages/data-context/src/actions/CohortsActions.ts

```diff
@@ -1,7 +1,11 @@
 import type { Cohort } from '@packages/types'
 import type { DataContext } from '..'
+import { WEIGHTED, WEIGHTED_EVEN } from '../util/weightedChoice'
+const debug = require('debug')('cypress:data-context:cohorts')
```

Comment:
Zach B schooled me on this a few days ago - we have [rules for how to namespace debug loggers](https://github.com/cypress-io/cypress/blob/develop/guides/debug-logs.md#choosing-a-namespace). This should probably be `cypress:data-context:actions:CohortActions`

Reply:
Thansk for pointing this out.  Will update